src/services/tokenService.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+
+const { Token } = require('../models/Token.js');
+
+async function save(userId, refreshToken) {
+  const token = await Token.findOne({ where: { userId } });
+
+  if (token) {
+    token.refreshToken = refreshToken;
+
+    await token.save();
+  }
+
+  await Token.create({
```

Comment:
The token creation logic is outside the condition block. This means a new token is created every time the save function is called, even if a token already exists for the user.